scripts/ci/podman-test.sh

```diff
@@ -7,7 +7,7 @@ export SKIP_CI_TEST=1
 
 cd ../../
 
-make install
+make install PREFIX=/usr
```

Comment:
@adrianreber I think we need to add `LIBDIR` as well as to install libcriu correctly

Reply:
I tried it in a container and on Ubuntu libcriu is installed in the correct location without explicitly setting LIBDIR.

Reply:
On Fedora 35, when I install criu with `make install PREFIX=/usr` libcriu is installed in:
```
ls -al /usr/lib/x86_64-linux-gnu/libcriu*
-rwxr-xr-x. 1 root root 292K May  2 10:08 /usr/lib/x86_64-linux-gnu/libcriu.a
lrwxrwxrwx. 1 root root   14 May  2 10:08 /usr/lib/x86_64-linux-gnu/libcriu.so -> libcriu.so.2.0
lrwxrwxrwx. 1 root root   14 May  2 10:08 /usr/lib/x86_64-linux-gnu/libcriu.so.2 -> libcriu.so.2.0
-rwxr-xr-x. 1 root root 194K May  2 10:08 /usr/lib/x86_64-linux-gnu/libcriu.so.2.0
```
However, the criu package for fedora installs these files in a different location:
```
rpm -ql criu-libs | grep libcriu
/usr/lib64/libcriu.so.2
/usr/lib64/libcriu.so.2.0
```
In this case crun would use `/usr/lib64/libcriu.so.2` instead of `/usr/lib/x86_64-linux-gnu/libcriu.so.2.0`.
Installing criu with the following command solved this problem:
```
make install PREFIX=/usr LIBDIR=/usr/lib64
```

Reply:
Yes, but the podman test is running on Ubuntu and on Ubuntu the default location is correct.

Reply:
@adrianreber Yes, you are right